site/buttons.js

```diff
@@ -1,15 +1,20 @@
-$('.simon-button.green').on('click', function() {
+const greenButton = document.querySelector(".simon-button.green");
```

Comment:
Thanks for doing this!

1. Run `npm run lint` and make sure style guide passes.
2. Define the variable right above where we bind the listener so that the proximity of the variable and where we use it reinforces the pattern we want for the student.

Reply:
One of my go-to examples, I was teaching a student and had code like:

```js
let myName = 'Jesse';
let myAge = 36;

console.log(`My name is ${myName} and my age is ${myAge});
```

They asked a good question: "Do the variables have to be called `myName` or `myAge`? Are those special?"

"No!" I said, "You can name them anything so long as you referencing using that name anywhere. Here, watch..." I then changed `myName` to `myWaffle` and `myAge` to `myGiraffe` (and updated the references).

The student: "Oh, I see! You can name variables anything so long as they begin with `my`."

(Not exaggerating even a little.)

Reply:
😆 the patterns that people pick up are interesting. I fixed linting errors and moved the usages next to the definitions.